# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xorg/xprint-nocshprofile.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur xc-vanilla/programs/Xserver/Xprint/etc/profile.d/Imakefile xc/programs/Xserver/Xprint/etc/profile.d/Imakefile
--- xc-vanilla/programs/Xserver/Xprint/etc/profile.d/Imakefile	2004-10-05 22:36:18.508466048 +0200
+++ xc/programs/Xserver/Xprint/etc/profile.d/Imakefile	2004-10-06 14:57:25.016588936 +0200
@@ -10,5 +10,4 @@
 
 all::
 
-InstallNamedProg(xprint.csh,xprint.csh,$(ETCDIR)/profile.d)
 InstallNamedProg(xprint.sh,xprint.sh,$(ETCDIR)/profile.d)
